Makefile

```diff
@@ -28,3 +28,17 @@ deploy-protocol 	:;	forge script script/DeployYieldNest.s.sol:DeployYieldNest --
 # Verify
 
 verify-roles 		:;	forge script script/Verify.s.sol --rpc-url ${rpc}
+
+# to set up cast wallet run:  cast wallet import yieldnestDeployerKey --interactive
+# then import the private key with which you wish to deploy, create a password and add the public address of that key to the .env under DEPLOYER_ADDRESS.
+
+# make ynEigen path=script/ynEigen/input/lsd-mainnet.json rpc=https://mainnet.infura.io/v3/your-key deployer=0xYourPublicAddressThatCorrespondsToYourSavedPrivateKeyInyieldnestDeployerKey api=etherscanApiKey
```

Comment:
we should not hardcode infura as an RPC anywhere at all

We use 3+ RPC providers

Reply:
fixed in [498985f](https://github.com/yieldnest/yieldnest-protocol/pull/158/commits/498985f751fe9f13299aaba4a6dd055950a650a6)